src/main/java/core/basesyntax/service/impl/ReadFromCsvFileImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFromCsvFileImpl implements FileReader {
+    private final String path;
```

Comment:
let's pass this as method parameter - utility classes should not store any state